chapters/SPDX-license-expressions.md

```diff
@@ -0,0 +1,252 @@
+
+For example, when one is required to comply with both the LGPL-2.1-only or MIT licenses, a valid expression would be:
+
+```text
+LGPL-2.1-only AND MIT
+```
+
+The "AND" operator is commutative, meaning that the above expression should be considered equivalent to:
+
+```text
+MIT AND LGPL-2.1-only
+```
+
+An example where all three different licenses apply would be:
+
+```text
+LGPL-2.1-only AND MIT AND BSD-2-Clause
+```
+
+### D.4.4 Additive "WITH" operator <a name="D.4.4"></a>
+
+Sometimes license texts are found with additional text, which might or might not modify the original license terms.
+
+In this case, use the binary "WITH" operator to construct a new license expression to represent the special situation. A valid `<license-expression>` is where the left operand is a `<simple-expression>` value and the right operand is a `<addition-expression>` that represents the additional text.
+
+The `<addition-expression>` can be either a `<license-exception-id>` from the SPDX License List, or a user defined addition reference in the form ["DocumentRef-"(idstring)":"]"AdditonRef-"(idstring)
+
+
+For example, when the Bison exception is to be applied to GPL-2.0-or-later, the expression would be:
+
+```text
+GPL-2.0-or-later WITH Bison-exception-2.2
+```
+
+The current set of valid license exceptions identifiers can be found in Annex [A.2](SPDX-license-list.md#A.2). For the most up to date set of exceptions please see [spdx.org/licenses](https://spdx.org/licenses).
+
+### D.4.5 Order of precedence and parentheses <a name="D.4.5"></a>
+
+The order of application of the operators in an expression matters (similar to mathematical operators). The default operator order of precedence of a `<license-expression>` a is:
+
+```text
++
+WITH
+AND
+OR
+```
+
+where a lower order operator is applied before a higher order operator.
+
+For example, the following expression:
+
+```text
+LGPL-2.1-only OR BSD-3-Clause AND MIT
+```
+
+represents a license choice between either LGPL-2.1-only and the expression BSD-3-Clause AND MIT because the AND operator takes precedence over (is applied before) the OR operator.
+
+When required to express an order of precedence that is different from the default order a `<license-expression>` can be encapsulated in pairs of parentheses: ( ), to indicate that the operators found inside the parentheses takes precedence over operators outside. This is also similar to the use of parentheses in an algebraic expression e.g., (5+7)/2.
+
+For instance, the following expression:
+
+```text
+MIT AND (LGPL-2.1-or-later OR BSD-3-Clause)
+```
+
+states the OR operator should be applied before the AND operator. That is, one should first select between the LGPL-2.1-or-later or the BSD-3-Clause license before applying the MIT license.
+
+### D.4.6 License expressions in RDF <a name="D.4.6"></a>
```

Comment:
This section needs updating, since the RDF are not the same anymore. (e.g., we have `spdx:ExpandedLicensing/ConjunctiveLicenseSet`, etc.)

But I'm fine with merging this and fixing it later.